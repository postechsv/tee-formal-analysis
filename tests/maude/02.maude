set show timing off .
set show advisories off .

load ../../formal-model/main

red 1 .