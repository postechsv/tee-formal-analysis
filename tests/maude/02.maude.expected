==========================================
reduce in CONVERSION : 1 .
rewrites: 0
result NzNat: 1
Bye.
